test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -72,7 +72,7 @@ public async Task GivenANewSearchParam_WhenReindexingComplete_ThenResourcesSearc
             }
             catch (FhirException ex) when (ex.StatusCode == HttpStatusCode.BadRequest && ex.Message.Contains("not enabled"))
             {
-                Skip.If(!_fixture.IsUsingInProcTestServer, "Reindex is not enabled on this server.");
+                // This test case won't run if reindex is not enabled
```

Comment:
Is the try catch block needed if this won't be hit?

Reply:
The attribute needs to be `[SkippableFact]` for Skip.If to work

Reply:
Updated to use the SkippableFact